hubspot/parse.go

```diff
@@ -76,3 +77,72 @@ func parsePagingNext(node *ajson.Node) (*ajson.Node, error) {
 
 	return next, nil
 }
+
+// getRecords returns the records from the response.
+func getRecords(node *ajson.Node) ([]map[string]interface{}, error) {
+	records, err := node.GetKey("results")
+	if err != nil {
+		return nil, err
+	}
+
+	if !records.IsArray() {
```

Comment:
You can use GetArray instead of `IsArray` and then `MustArray`: https://pkg.go.dev/github.com/spyzhov/ajson#Node.GetArray

Reply:
Moving this to the next PR along with changes to getRecords